server/src/routes/api/v1/showsRouter.js

```diff
@@ -0,0 +1,23 @@
+import express from "express"
+
+import ShowsClient from "../../../apiClient/ShowsClient"
+
+const showsRouter = new express.Router()
+
+showsRouter.get("/", async (req, res) =>{
+  const pageNumber = req.query.pageNumber
+
+  try{
+    const showsResponse = await ShowsClient.getShows(pageNumber)
+    const showData = JSON.parse(showsResponse)
```

Comment:
ideally line 12 should be moved into the ShowsClient, which can be responsible for parsing the data in addition to making a request to the api